### ISSUE TITLE
Host build / testbench docs

### ISSUE THREAD

**Issue by jajanusz** [thesofproject/sof-docs#97]
There is almost no info about testbench in docs (only how to build it).
We need description telling what it is and how to use it.

**Comment by jajanusz** [comment]
Probably @singalsu is the best person to address this

**Comment by lgirdwood** [comment]
@jajanusz yeah I agree. @singalsu I think we need some simple introduction and instruction for using the testbench (with say the SRC), e.g. just think of realworld usecase and detail the steps.

**Comment by singalsu** [comment]
I have some user's guide text written for SRC. I can extract the test bench part into sof-docs pretty quickly. Also the rest would be useful to publish.

**Comment by lgirdwood** [comment]
@singalsu sounds good to me.

**Comment by singalsu** [comment]
I haven't had time for this but stay tuned.

**Comment by jajanusz** [comment]
@singalsu You can wait for refactor first, to not write it twice: https://github.com/thesofproject/sof/issues/1229
